Adds note about color equality after wide gamut. (#11132)
flutter/website: src/content/release/breaking-changes/wide-gamut-framework.md

final x = color.withValues(alpha: 0.0);
```

### Equality

> Now that Color will be storing its color components as floating-point numbers
> equality will work slightly different. When calculating colors there may be tiny
> difference in numbers which could be considered equal. To accommodate this use
> the [`closeTo`][] matcher or the [`isColorSameAs`][] matcher.

Reviewer: ```suggestion
Now that `Color` is storing its color components as floating-point numbers,
equality work slightly differently.
When calculating colors there might be tiny
difference in values that could be considered equal.
To accommodate this use the [`closeTo`][] matcher or the [`isColorSameAs`][] matcher.
```
